src/utils/generate-pdf.js

```diff
@@ -69,16 +72,24 @@ const generateBill = (bill, date) => {
   doc.rect(165, 110 + 2 + totalOffset, 30, 7);
   boldText(doc, bill.expensesTotal, 195 - 2, 115 + 2 + totalOffset, { align: 'right' });
 
-  boldText(doc, 'LIITTEET', 15, 260);
-  doc.text(`Kappaletta: ${bill.attachments.length}`, 15, 265);
+  boldText(doc, 'LIITTEET', 15, 250);
+  doc.text(`Kappaletta: ${bill.attachments.length}`, 15, 255);
+
+  doc.rect(15, 260, 90, 12);
+  doc.text('IBAN:', 15 + 2, 265);
+  doc.text(bill.biller.iban, 15 + 2, 270);
+
+  doc.rect(105, 260, 90, 12);
+  doc.text('Yhteensä EUR:', 105 + 2, 265);
+  doc.text(bill.expensesTotal, 105 + 2, 270);
 
-  doc.rect(15, 270, 90, 12);
-  doc.text('IBAN:', 15 + 2, 275);
-  doc.text(bill.biller.iban, 15 + 2, 280);
+  let canvas = createCanvas();
+  let context = canvas.getContext('2d');
+  writeOnCanvas(canvas, getSerialNumber(bill.biller.iban, bill.expensesTotal, "3", new Date(2019, 2, 2)));
+  context.fill();
 
-  doc.rect(105, 270, 90, 12);
-  doc.text('Yhteensä EUR:', 105 + 2, 275);
-  doc.text(bill.expensesTotal, 105 + 2, 280);
+  var img = canvas.toDataURL("image/jpeg", 1.0);
```

Comment:
The pdf generator adds _images_ as attachments so should be better called `barcode` for clarity. If possible it should also be a `png` file (not sure if the suggestion is correct) as the format works best with solid-color images. Best case it should be an `svg` image.
```suggestion
  var barcode = canvas.toDataURL("image/png", 1.0);
```